index.html

```diff
@@ -62,15 +66,14 @@
 
 * [FedCM](https://github.com/w3ctag/design-reviews/issues/718). Third-party cookies have been used to support single sign-on. FedCM is a set of technologies built to support identity federation, without replicating all functionality of third-party cookies.
 * [CHIPS](https://github.com/w3ctag/design-reviews/issues/654). The goal of CHIPS is to allow state, without supporting correlation or tracking between web sites which don't know they are collaborating, for example, when embedding third-party services like customer service webchat.
-* [Fenced Frames](https://github.com/w3ctag/design-reviews/issues/735). This provides a new top-level browsing context which enables content from multiple parties to be embedded on a page, without the embedding site being able to communicate with the embedded site.
 
 In all cases, it is important to continually review how such proposals might interact with other emerging or proposed APIs. Be aware that a set of new technologies which carry minimal risk individually, could be used in combination for tracking or profiling of web users.
 
 ## Leaving the web better than we found it
 
 We support deprecating third-party cookies from the web platform, and we embrace the opportunity to improve the privacy features of the web. When we review new technologies to replace third-party cookies, we need to ensure that the replacements do not recreate the same pitfalls to privacy.
 
-The TAG considers each new technology proposal *both* individually, *and* as they fit together with the web platform as a whole. The web must be cross-platform, so multi-stakeholder (in particular, multi-browser) support for privacy-related specifications is essential if they are going to achieve the goal of increasing privacy on the web. When we consider whether something makes the web platform better, we should be explicit about what the baseline for comparison is. Is a proposal better for privacy when compared to usage of third-party cookies? Or when compared with a web free from third-party cookies altogether? What about when some user agents restrict third-party cookies, but others do not?
+The TAG considers each new technology proposal *both* individually, *and* as they fit together with the web platform as a whole. The web must be cross-platform, so multi-implementer (multi-browser) support and developer support for privacy-related specifications is essential if they are going to achieve the goal of increasing privacy on the web. When we consider whether something makes the web platform better, we should be explicit about what the baseline for comparison is. Is a proposal better for privacy when compared to usage of third-party cookies? Or when compared with a web free from third-party cookies altogether? What about when some user agents restrict third-party cookies, but others do not?
```

Comment:
Why this change?  It doesn't seem relevant to the main thrust of the change (and I think that it's a regression).

Reply:
Clears up the language around implementer vs stakeholder. I've had feedback from many quarters that when we mean "multi implementer" we should say implementer rather than simply "stakeholder". So I was going to say "multi-implementer and multi-stakeholder" but I realised iit would be clear to say "multi-implementer and developer"...